.eslintrc.json

```diff
@@ -19,6 +19,10 @@
   },
   "plugins": ["react"],
   "rules": {
-    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "no-underscore-dangle":"off",
+    "import/no-cycle": "off",
+    "react/forbid-prop-types": "off",
+    "linebreak-style": "off"
```

Comment:
It should be on

Reply:
turned on all, except "react/forbid-prop-types", because then eslint not allow comlipe with Object prop-type